src/main/java/codeu/model/store/basic/UserStore.java

```diff
@@ -121,4 +121,18 @@ public boolean isUserRegistered(String username) {
   public void setUsers(List<User> users) {
     this.users = users;
   }
+
+  /** Update user whenever about me is updated. */
+  public void updateUser(User newUser, String username, String aboutMe) {
+    for (User user : users) {
+      if (user.getName().equals(username)) {
+        users.remove(user);
+        users.add(newUser);
+        persistentStorageAgent.update(aboutMe, username);
+        return;
+      }
+    }
+
+
+  }
```

Comment:
is there a way to do this without removing the user?

Reply:
I've been thinking about it for a while, but haven't found a solution yet. You have an idea?